other/bootstrap_daemon/tox_bootstrap.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Tox DHT Bootstrap Daemon
+After=network.target
+
+[Service]
+Type=forking
+PermissionsStartOnly=true
+ExecStartPre=/usr/bin/touch /var/run/tox-bootstrapd.pid
+ExecStartPre=/usr/bin/chown tox-bootstrapd:tox-bootstrapd -R /var/run/tox-bootstrapd.pid
```

Comment:
Systemd cannot work with relative paths (according to it's docs). I think, "/bin/chown" etc - it's more common, let it be dafault, and I added to doc one line about it.